api_tests/nodes/serializers/test_serializers.py

```diff
@@ -67,13 +65,12 @@ def test_fork_serialization(self):
         # Relationships
         relationships = data['relationships']
         forked_from = relationships['forked_from']['links']['related']['href']
-        assert_equal(
-            urlparse(forked_from).path,
-            '/{}nodes/{}/'.format(API_BASE, node._id)
-        )
+        assert (
+            urlparse(forked_from).path ==
+            '/{}nodes/{}/'.format(API_BASE, node._id))
 
-    def test_template_serialization(self):
-        node = NodeFactory(creator=self.user)
+    #   test_template_serialization
+        node = NodeFactory(creator=user)
         fork = node.use_as_template(auth=Auth(user=node.creator))
```

Comment:
I get this error when I try to do that:
E   AttributeError: 'tuple' object has no attribute 'user'